doc/source/whatsnew/v0.25.0.rst

```diff
@@ -388,6 +388,7 @@ Groupby/Resample/Rolling
 - Ensured that result group order is correct when grouping on an ordered ``Categorical`` and specifying ``observed=True`` (:issue:`25871`, :issue:`25167`)
 - Bug in :meth:`pandas.core.window.Rolling.min` and :meth:`pandas.core.window.Rolling.max` that caused a memory leak (:issue:`25893`)
 - Bug in :func:`idxmax` and :func:`idxmin` on :meth:`DataFrame.groupby` with datetime column would return incorrect dtype (:issue:`25444`, :issue:`15306`)
+- Bug in :func:`cumsum`, :func:`cumprod`, :func:`cummin` and :func:`cummax` on :meth:`DataFrame.groupby` with categorical column having absent categories, would return incorrect result or segfault (:issue:`16771`)
```

Comment:
this wont' render correctly, so you can just use ``cumprod`` OR use a fully qualified name, e.g. ``:meth:`pandas.core.groupby.Groupby.cumsum` ``; i think the prior whatsnew is wrong as well